<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Model_3D_display</class>
 <widget class="QWidget" name="Model_3D_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>295</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hor_3D">
    <item>
     <widget class="QPushButton" name="btn_selectFile">
      <property name="styleSheet">
       <string notr="true">QLabel {
	border: none;
	font-size: 18px;
}</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_show3D">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font-size: 18px;	
	color: #039BE5;
}</string>
      </property>
      <property name="text">
       <string>3D Show</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hor_file">
    <item>
     <widget class="QLabel" name="label_filename">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border: none;
	font-size: 18px;
}</string>
      </property>
      <property name="text">
       <string>FileName</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="value_filePath"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="value_Ny">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>120</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ny">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>120</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ny:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_Nz">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>120</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_Nx">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>120</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ny_2">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>120</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Nx:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Nz">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Nz:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
